.gitignore

```diff
@@ -1 +1,6 @@
-vendor
+# Ignoring directories.
+/.idea/
```

Comment:
`.idea` seems to be coming from jetbrains, which is a personal folder. Couldn't you just not add this folder to git as well? Fine for me

Reply:
I don't entirely agree with you, but it's okay.